caf-logging-serilog/official-build.props

```diff
@@ -0,0 +1 @@
+SEPG_BUILD_ENV_IMAGE=cafapi/buildenv-dotnet:1.0.0
```

Comment:
Having an `official-build.props` file at this level can't be doing anything.  SEPG will only pay attention to the repository-level one.